src/DynamoCLI/CommandLineRunner.cs

```diff
@@ -46,7 +46,7 @@ private static XmlDocument RunCommandLineArgs(DynamoModel model, StartupUtils.Co
                 Console.WriteLine("geometryFilePath option is only available when running DynamoWPFCLI, not DynamoCLI");
             }
 
-            model.HostAnalyticsInfo = cmdLineArgs.AnalyticsInfo;
+            DynamoModel.HostAnalyticsInfo = cmdLineArgs.AnalyticsInfo;
```

Comment:
I think since we're at 3.0 we should just unify these properties, either use the `Info` or use the properties on DynamoModel.